pyre/bench/synth/foriter_call_body.py

```
@@ -38,3 +45,29 @@ def accumulating():
 print(accumulating())
 # Expected: sum(n for n in range(100)) * sum(j for j in range(200))
 #         = 4950 * 19900 = 98505000
+
+
+class NextIter:
+    def __init__(self, n):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)